dtfs-central-api/src/types/utilisation-reports.ts

```diff
@@ -108,11 +83,6 @@ export type NewPaymentDetails = {
   reference?: string;
 };
 
-export type FeeRecordToKey = FeeRecordItem & {
-  paymentsReceived: CurrencyAndAmount[];
-  status: FeeRecordStatus;
-};
-
 export type UtilisationReportWithFeeRecordsToKey = {
```

Comment:
I've only just processed this name sorry for not noting earlier, but I would be tempted to name this just 'FeeRecordsToKey' or something similar with the 'FeeRecordsToKey' being the key "thing", and then renaming the id field to 'reportId'. The reason being that this type isn't really a report with something added, it's mostly just some fee records mapped into an interesting shape with a few bonus pieces of information about the report really.  'With' names can get quite confusing as you get more of them and have to keep checking which one you should be using at any given time and what the difference is between them.